### DISCUSSION TITLE
How to find and use LLVM libraries?

### DISCUSSION THREAD

**Discussion by ghost** [JulNadeauCA/BSDBuild#10] [General]
With CMake, it's very simple: `find_package(LLVM REQUIRED)`

**Comment by JulNadeauCA** [comment]
You would have to write a test module for it. Which LLVM libraries are you looking for specifically?

`BSDBuild/lldb.pm` would be a good starting point since it already looks for a `llvm-config`, you would just need to adapt it to return the libraries you need, and add a `CMAKE_lldb()` routine to use `find_package(LLVM)` in the CMake case.
